### ISSUE TITLE
Swiping the cards it is removing entire layout

### ISSUE THREAD

**Issue by phani51** [Arjun-sna/android-swipecards-view#4]
Hi,
I have an issue while swiping the cards, i.e, when i swiping the cards entire layout is going.

**Comment by Arjun-sna** [comment]
do you mean that all cards in the layout move on swiping?

**Comment by phani51** [comment]
No,when i try to swipe one card, that entire cards are going (for example i have 5 cards in the layout when i try to swipe 1st card then all the other cards are going and nothing to show in the layout).
